contrib/vagrant/start.sh

```diff
@@ -84,17 +84,21 @@ function write_footer() {
     ipv6_addr="${2}"
     filename="${3}"
 
-    if [ -n "${IPV4}" ]; then
+    if [ -n "${IPV4}" ] && [ "${IPV4}" -ne "0" ]; then
         ipv4_options="--ipv4 --ipv4-range 10.${index}.0.1 "
     fi
 
+    if [ -n "${K8S}" ]; then
```

Comment:
True but `IPV4` is set to `1` but default. Should I change `if [ -n "${K8S}" ]; then` to `if [ -n "${K8S}" ] && [ "${K8S}" -ne "0" ]; then`? and the `NFS` as well?